### PR TITLE
Add a way to use cprofile to generate stats and render as flamegraph

### PR DESCRIPTION
**Pull request by kimishpatel**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #1225

Summary:
While trying to debug slow export I found using cprofile quite useful.
However, it is much more intuitive to use icicle kind of flamegraphs.
This PR adds this util by building on top of snakeviz

Test Plan:
Tested locally

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D51380270](https://our.internmc.facebook.com/intern/diff/D51380270)

### CODE REVIEW

**Comment by kimishpatel** [comment]
@kimishpatel has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51380270).

**Comment by kimishpatel** [comment]
@kimishpatel has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51380270).

**Comment by kimishpatel** [comment]
@kimishpatel has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51380270).

File: util/cprofiler_flamegraph.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Requirements
+
+CProfilerFlameGraph depends on snakeviz to render flamegraph as html file. Main snakeviz repo requires renders flamegraph by launching webserver and rendering html through that. On some enviornments you want to just get a statically renderable html that can be shared. Snakeviz's author provides utility for this at https://gist.github.com/jiffyclub/6b5e0f0f05ab487ff607. This util just combined the patch from gist and added a util that does not need to launch server.
+
+## Installation
+
+pip install snakeviz
```
Comment (mcr229) [inline util/cprofiler_flamegraph.md:7]: Create an install_requirements.sh for this?

Comment (kimishpatel) [reply util/cprofiler_flamegraph.md:7]: It is just snakeviz pip install, so didnt quite do it.

**Comment by kimishpatel** [comment]
@kimishpatel has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51380270).

**Comment by kimishpatel** [comment]
@kimishpatel has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51380270).

**Comment by kimishpatel** [comment]
@kimishpatel has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51380270).

**Comment by kimishpatel** [comment]
@kimishpatel has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51380270).

**Comment by kimishpatel** [comment]
@kimishpatel has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51380270).

**Pull request closed without merging**